static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -203,7 +203,7 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
     color: 'green300',
     description: stripOrigin(frame.description),
     tabKey: TabKey.NETWORK,
-    title: 'Navigation',
+    title: 'Navigation (Push)',
```

Comment:
Was this changed in a design mock?

Reply:
No, but there are two Navigation frame types with the same title which was causing issues in the type selector (there were two 'Navigation'). I discussed this with Ryan and we decided to change the title for one of them